config/application.rb

```diff
@@ -100,6 +100,10 @@ class Application < Rails::Application
       end
     end
 
+    if GalleryConfig.disable_ssl_verify
+      OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
+    end
```

Comment:
Plan on merging that other branch first, so this won't be a diff for this merge.